django_project/frontend/src/pages/Dashboard/MapLibre/LayerType/RasterCog.jsx

```diff
@@ -56,26 +57,35 @@ export default function rasterCogLayer(map, id, data, setData, contextLayerData,
       if (!colors.length) {
         return
       }
+
+      let classifications = [];
+
+      await DjangoRequests.post(
+    `/api/style/raster-classification`, {
```

Comment:
Please change to `/api/raster/distribution`
So it is just return the distributions and we can reuse the API